spec/dummy/db/migrate/001_add_test_to_source_records.rb

```diff
@@ -0,0 +1,7 @@
+class AddTestToSourceRecords < ActiveRecord::Migration
+  def self.up
```

Comment:
instead of separate `up` and `down` methods, just `def change` instead, especially since we're not actually doing anything. See http://edgeguides.rubyonrails.org/active_record_migrations.html#migration-overview and http://edgeguides.rubyonrails.org/active_record_migrations.html#using-the-up-down-methods.